src/components/NcAppSidebar/NcAppSidebarTabs.vue

```diff
@@ -84,6 +84,13 @@ export default {
 			type: String,
 			default: '',
 		},
+		/**
+		 * Force the tab navigation to display even if there is only one tab
+		 */
+		forceNavigation: {
+			type: Boolean,
+			default: false,
+		},
```

Comment:
I'd call it `forceTabs` or `showSingleTab`, because there is nothing e called "navigation" in this component including docs.